README.md

```diff
@@ -115,12 +125,12 @@ You can run the collection's test suites with the commands:
 The current process for publishing new versions of the Windows Core Collection is manual, and requires a user who has access to the `ansible` namespace on Ansible Galaxy and Automation Hub to publish the build artifact.
 
   1. Update the CHANGELOG:
-    1. Make sure you have [`antsibull-changelog`](https://pypi.org/project/antsibull-changelog/) installed.
-    1. Make sure there are fragments for all known changes in `changelogs/fragments`.
-    1. Run `antsibull-changelog release`
-  1. Update `galaxy.yml` with the new `version` for the collection.
-  1. Create a release in GitHub to tag the commit at the version to build.
-  1. Run the following commands to build and release the new version on Galaxy:
+    1. Make sure you have [`antsibull-changelog`](https://pypi.org/project/antsibull-changelog/) installed.  
+    2. Make sure there are fragments for all known changes in `changelogs/fragments`.  
```

Comment:
Isn't it simpler to keep them at 1. Makes it easier to add/change/remove entries without re-ordering the entire list.

Reply:
1, 1. 1. doesn't make sense from a docs perspective. If you feel strongly that you don't want to deal with renumbering, we should use bullet items instead. (tho I can't promise I won't forget 3 months from now and try to number 'em again.. it's a docs thing :-)

Reply:
The raw markdown shows 1. 1. 1. but the actual markdown output orders them properly. Sounds like a no-brainer to have something do it automatically for you.

Reply:
But I'm also happy them being dot points, I just copied what was from the crypto README.

Reply:
something going wonky on my side. gonna back out some commits so bear with a moment or two